### ISSUE TITLE
ModuleNotFoundError: No module named 'gpt_do.vendor'

### ISSUE THREAD

**Issue by xgarrido** [yasyf/gpt-do#1]
I have installed `gpt-do` with `pipx` and when using `--model=chatgpt` the scripts raises an error 

```
$ ddo echo hello to file hello.txt --model=chatgpt                                                                                    
Traceback (most recent call last):
  File "/home/tester/.local/bin/ddo", line 8, in <module>
    sys.exit(do())
  File "/home/tester/.local/pipx/venvs/gpt-do/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/tester/.local/pipx/venvs/gpt-do/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/tester/.local/pipx/venvs/gpt-do/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/tester/.local/pipx/venvs/gpt-do/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/tester/.local/pipx/venvs/gpt-do/lib/python3.10/site-packages/gpt_do/cli.py", line 37, in do
    do = get_doer(model)(debug=debug)
  File "/home/tester/.local/pipx/venvs/gpt-do/lib/python3.10/site-packages/gpt_do/cli.py", line 6, in get_doer
    from gpt_do.doers.pywright_doer import PywrightDoer
  File "/home/tester/.local/pipx/venvs/gpt-do/lib/python3.10/site-packages/gpt_do/doers/pywright_doer.py", line 2, in <module>
    from gpt_do.vendor.chatgpt_wrapper.chatgpt_wrapper import ChatGPT
ModuleNotFoundError: No module named 'gpt_do.vendor'
```

The `vendor` directory is actually missing (I guess the `git submodule` is not populated by `pipx`)

**Comment by yasyf** [comment]
oops! Looking into this

**Comment by yasyf** [comment]
@xgarrido should be fixed in `0.1.15`!

**Comment by yasyf** [comment]
But note that you are likely to run into mmabrouk/chatgpt-wrapper#35
